srcpkgs/seq24/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'seq24'
+pkgname=seq24
+version=0.9.3
+revision=1
+build_style=gnu-configure
+configure_args="--disable-lash"
+hostmakedepends="pkg-config alsa-lib-devel automake"
+makedepends="gtkmm2-devel alsa-lib-devel jack-devel"
```

Comment:
We should be trying to get rid of gtk2 packages, not add more of them. Is there a plan to support gtk3/4 or move to another toolkit?

Reply:
uh...understand... I recently used this to resurrect some old big midi sessions, it worked very well so I shared my template. 
It's an 'old' sequencer, I think there are not many future plans but we'll see. Thanks.

Reply:
Even if we don't end up merging this PR, the template is still here and interested people can build it on their own, so thanks for that!